custom-login/extra-webpack.config.js

```diff
@@ -0,0 +1,28 @@
+// Read environment variables from "testenv" file in root of the repository
+const path = require('path');
+require('dotenv').config({
+  path: path.resolve(__dirname, '..', 'testenv')
+});
+
+const webpack = require('webpack');
+const env = {};
+
+// List of environment variables made available to the app
+[
+  'ISSUER',
+  'CLIENT_ID',
+].forEach(function (key) {
+  if (!process.env[key]) {
+    throw new Error(`Environment variable ${key} must be set. See README.md`);
```

Comment:
Since you've told them to put these in a file, would the mention of environment variables be confusing?

Reply:
I think I will change the wording in the README so it's more clear they CAN put the environment variables in a file. Some developers would choose to not do this, particularly if they are working with more than one clientId or issuer.